infra/es_cluster.tf

```diff
@@ -148,6 +160,15 @@ apt-get -y upgrade
 curl -sSL https://dl.google.com/cloudagents/install-logging-agent.sh | bash
 
 ## Install Docker
+
+mkdir -p /etc/docker
+cat <<CONFIG > /etc/docker/daemon.json
+{
+  "log-driver": "json-file",
+  "log-opts": {"max-size": "10m", "max-file": "3"}
+}
+CONFIG
```

Comment:
This adds log rotation to the Docker daemon, which I think may help because sometimes ES just prints _a lot_.